### PR TITLE
Feature/chat list landscape UI

### PR DESCRIPTION
**Pull request by fpohtmeh-github**

### CODE REVIEW

File: src/controllers/MessagesController.cpp [hunk 1]
```diff
@@ -73,15 +73,21 @@ Self::MessagesController(Messenger *messenger, const Settings *settings, Models
     connect(m_messenger, &Messenger::updateMessage, this, &Self::onUpdateMessage);
 }
 
-void Self::loadMessages(const ChatHandler &chat)
+void Self::loadMessages(const ChatHandler &chat, bool isNewChat)
```
Comment (SergeySeroshtan) [inline src/controllers/MessagesController.cpp]: Replace boolean flag with 2 methods.

File: include/controllers/MessagesController.h [hunk 2]
```diff
@@ -52,7 +52,7 @@ class MessagesController : public QObject
     MessagesController(Messenger *messenger, const Settings *settings, Models *models, UserDatabase *userDatabase,
                        QObject *parent);
 
-    void loadMessages(const ChatHandler &chat);
+    void loadMessages(const ChatHandler &chat, bool isNewChat);
```
Comment (SergeySeroshtan) [inline include/controllers/MessagesController.h]: Replace boolean flag with 2 methods.
Is this method still in use?

File: include/controllers/ChatsController.h [hunk 3]
```diff
@@ -70,7 +70,7 @@ class ChatsController : public QObject
     void createChatWithUserId(const UserId &userId);
     void createGroupChat(const QString &groupName, const Contacts &contacts);
 
-    void openChat(const ModifiableChatHandler &chat);
+    void openChat(const ModifiableChatHandler &chat, bool isNew);
```
Comment (SergeySeroshtan) [inline include/controllers/ChatsController.h]: Replace boolean flag with 2 methods.

File: src/controllers/MessagesController.cpp (hunk 1 above)
Comment (fpohtmeh-github) [reply src/controllers/MessagesController.cpp]: Splitted into 2 methods: loadChat & loadNewChat

File: include/controllers/ChatsController.h (hunk 3 above)
Comment (fpohtmeh-github) [reply include/controllers/ChatsController.h]: Replaced with openChat & createChat

File: include/controllers/MessagesController.h (hunk 2 above)
Comment (fpohtmeh-github) [reply include/controllers/MessagesController.h]: Replaced and renamed.
All methods for loading are used

**Comment by fpohtmeh-github** [comment]
@SergeySeroshtan please review the latest commit and merge

**Review by SergeySeroshtan — APPROVED** [review]
Looks much better )

**Pull request merged**